__tests__/intergration.test.js

```diff
@@ -120,7 +120,7 @@ describe("GET /api/articles/:id/comments", () => {
         })
     })
 })
-describe("GET /api/articles/", () => {
+describe.only("GET /api/articles/", () => {
     test("Should return a 200 status code", () => {
         return request(app).get("/api/articles/").expect(200);
     })
```

Comment:
Don't forget to remove any .onlys before pushing - this will only be checking this test in the linter before pushing.